libminifi/test/TestBase.cpp

```diff
@@ -150,7 +139,7 @@ std::shared_ptr<core::Processor> TestPlan::addProcessor(const std::string &proce
 }
 
 std::shared_ptr<core::Processor> TestPlan::addProcessor(const std::string &processor_name, const std::string &name, const std::initializer_list<core::Relationship>& relationships,
-                                                        bool linkToPrevious) {
+  bool linkToPrevious) {
```

Comment:
Continuation indentation should be 4 spaces

Reply:
That is correct, updated.